draft-ietf-rtcweb-jsep.xml

```diff
@@ -1534,8 +1534,8 @@ candidate:1 1 UDP 1694498815 192.0.2.33 10000 typ host
           with these exceptions: <list style="symbols">
               <t>The "s=" and "t=" lines MUST stay the same.</t>
 
-              <t>Each "m=" and c=" line MUST be filled in with the port and
-              address of the default candidate for the m= section, as described
+              <t>Each "m=" and c=" line MUST be filled in with the port, protocol,
+              and address of the default candidate for the m= section, as described
```

Comment:
Is the implication here that the protocol may change from
UDP/TLS/RTP/SAVPF to TCP/TLS/RTP/SAVPF? If so that would be good to state explicitly/